gcloud-java-core/src/main/java/com/google/gcloud/AuthCredentials.java

```diff
@@ -131,7 +136,7 @@ public String account() {
       @Override
       public byte[] sign(byte[] toSign) {
         try {
-          Object signingResult = signForApp.invoke(appIdentityService, (Object) toSign);
+          Object signingResult = signForApp.invoke(appIdentityService, toSign);
```

Comment:
My IDE auto removes the `(Object)` cast. I can revert this change, but the cast seems unnecessary so I kept the change.